tests/testthat/test-app_pws.R

```diff
@@ -0,0 +1,300 @@
+library(shinytest2)
+
+brush_vals <- c(
+  1.575922029932, 2.2959573557845,
+  1.218473025872, 2.2972720836904, 252, 532, 251, 349, 252, 532, 251, 349, 1,
+  1, "1:5", "1:5", 1, 5, 1, 5, 28.0413099315069, 1583.52054794521, 368.846473150198,
+  5.47945205479452, character(0), character(0), "xy", "plot_with_settings-plot_brush",
+  "plot_with_settings-plot_main"
+)
+
+hover_vals <- c(
+  2.95684692272772, 2.34130469829519,
+  789, 247, 789, 247, 1, 1, "1:5", "1:5", 1, 5, 1, 5, 28.0413099315069, 1583.52054794521,
+  368.846473150198, 5.47945205479452, character(0), character(0)
+)
+
+dbl_click_vals <- c(
+  2.95684692272772, 2.34130469829519,
+  789, 247, 789, 247, 1, 1, "1:5", "1:5", 1, 5, 1, 5, 28.0413099315069, 1583.52054794521,
+  368.846473150198, 5.47945205479452, character(0), character(0)
+)
+
+click_vals <- c(
+  2.95941847746291, 1.9450111668517,
+  790, 283, 790, 283, 1, 1, "1:5", "1:5", 1, 5, 1, 5, 28.0413099315069, 1583.52054794521,
+  368.846473150198, 5.47945205479452, character(0), character(0)
+)
+
+# one app opens for all tests
+pws <- app_pws()
+app <- AppDriver$new(pws, name = pws, height = 937, width = 1619)
+
+# tests plot click functionalities
+testthat::test_that("plot_with_settings: click functionalities ggplot2", {
```

Comment:
This test has a wrong name, should be:
`plot_with_settings: returns a reactive consistent with inputs configuration`